src/main/webapp/script.js

```diff
@@ -0,0 +1,31 @@
+function loadBody() {
+  console.log("Verifying login...");
+  verifyLogin();
+}
+
+/** Contacts Login servlet to verify if user is logged in. */
+function verifyLogin() {
+  console.log("Fetching login status...");
+  fetch('/login').then(response => response.json()).then((arrStrings) => {
+    const loginElement = document.getElementById("login-box");
+    if (arrStrings[0] == "false") {
+      document.getElementById("school-name").style.display = "none";
+    } else {
+      const schoolName = document.getElementById("school-name");
+      schoolName.style.display = "block";
+      const start = arrStrings[1].indexOf('@');
```

Comment:
This isn't a great way to figure out if the user's email is valid. This kind of into should be returned from the server specifically, not packaged inside some html.

Reply:
Ditto for school.

Reply:
I want to use OAuth for this! However, I can't enable it right now - Travis mentioned he's going to look into it and I'm going to try again now that we have our project ID. I just wanted to have a rudimentary version working so I had some form of error checking (even though it's not great)

Reply:
^also Laura mentioned the trumail API which I'll look into